### PR TITLE
Issues Configuration

### PR DESCRIPTION
**Pull request by tedpatrick**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```
Comment (marimeireles) [inline .github/ISSUE_TEMPLATE/config.yml:1]: I guess @antocuni asked specifically to be able to create empty templates for having some freedom when he was creating issues that didn't fit the other categories. Wondering if you still have opinions on this Antonio or fine with discussions?

File: .github/stale.yaml [hunk 2]
```diff
@@ -1,26 +0,0 @@
-# Number of days of inactivity before an issue becomes stale
```
Comment (marimeireles) [inline .github/stale.yaml:1]: We're dropping the stale stuff?

**Review by marimeireles — COMMENTED** [review]
Thanks for this Ted!

File: .github/stale.yaml (hunk 2 above)
Comment (tedpatrick) [reply .github/stale.yaml:1]: Is we are actively categorizing/closing issues, I do not think this is needed.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (tedpatrick) [reply .github/ISSUE_TEMPLATE/config.yml:1]: I think the variety of Github Discussion categories should appease Antonio (⛷️ ). Let's ship it, it is just a boolean away to re-enable.

**Review by fpliger — APPROVED** [review]
🚀

**Pull request merged**